{"name": "Page Title Test Case 001", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for chrome using Selenium Manager.; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location", "trace": "service = <selenium.webdriver.chrome.service.Service object at 0x000002DBD6F5BE50>\noptions = <selenium.webdriver.chrome.options.Options object at 0x000002DBD9746CD0>\n\n    @staticmethod\n    def get_path(service: Service, options: BaseOptions) -> str:\n        path = service.path\n        try:\n>           path = SeleniumManager().driver_location(options) if path is None else path\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nvenv\\Lib\\site-packages\\selenium\\webdriver\\common\\selenium_manager.py:90: in driver_location\n    output = self.run(args)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ['F:\\\\Credence- CT14 batch\\\\Python-Automation (July_Month)\\\\Framework - Tushar Sir\\\\LogGeneration - CredKart_Pytest_Pr...\\site-packages\\\\selenium\\\\webdriver\\\\common\\\\windows\\\\selenium-manager.exe', '--browser', 'chrome', '--output', 'json']\n\n    @staticmethod\n    def run(args: List[str]) -> dict:\n        \"\"\"\n        Executes the Selenium Manager Binary.\n        :Args:\n         - args: the components of the command being executed.\n        :Returns: The log string containing the driver location.\n        \"\"\"\n        if logger.getEffectiveLevel() == logging.DEBUG:\n            args.append(\"--debug\")\n        args.append(\"--output\")\n        args.append(\"json\")\n    \n        command = \" \".join(args)\n        logger.debug(f\"Executing process: {command}\")\n        try:\n            if sys.platform == \"win32\":\n                completed_proc = subprocess.run(\n                    args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, creationflags=subprocess.CREATE_NO_WINDOW\n                )\n            else:\n                completed_proc = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)\n            stdout = completed_proc.stdout.decode(\"utf-8\").rstrip(\"\\n\")\n            stderr = completed_proc.stderr.decode(\"utf-8\").rstrip(\"\\n\")\n            output = json.loads(stdout)\n            result = output[\"result\"]\n        except Exception as err:\n            raise WebDriverException(f\"Unsuccessful command executed: {command}\") from err\n    \n        for item in output[\"logs\"]:\n            if item[\"level\"] == \"WARN\":\n                logger.warning(item[\"message\"])\n            if item[\"level\"] == \"DEBUG\" or item[\"level\"] == \"INFO\":\n                logger.debug(item[\"message\"])\n    \n        if completed_proc.returncode:\n>           raise WebDriverException(f\"Unsuccessful command executed: {command}.\\n{result}{stderr}\")\nE           selenium.common.exceptions.WebDriverException: Message: Unsuccessful command executed: F:\\Credence- CT14 batch\\Python-Automation (July_Month)\\Framework - Tushar Sir\\LogGeneration - CredKart_Pytest_Project 05 Aug 2023\\Project-OrangeHRM\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\windows\\selenium-manager.exe --browser chrome --output json.\nE           {'code': 65, 'message': 'error sending request for url (https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json): error trying to connect: An existing connection was forcibly closed by the remote host. (os error 10054)', 'driver_path': '', 'browser_path': ''}\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\common\\selenium_manager.py:138: WebDriverException\n\nThe above exception was the direct cause of the following exception:\n\nbrowser = None\n\n    @pytest.fixture\n    def setup(browser):\n            # options = Options()\n            # options.binary_location = 'C:\\\\Program Files\\\\Mozilla Firefox\\\\firefox.exe'\n            # driver = webdriver.Firefox(options=options)\n        if browser == \"Chrome\":\n            driver = webdriver.Chrome()\n        elif browser == \"Edge\":\n            driver = webdriver.Edge()\n        elif browser == \"Safari\":\n            driver = webdriver.Safari()\n        else:\n            win_options = webdriver.ChromeOptions()\n            win_options.add_argument(\"headless\")\n>           driver = webdriver.Chrome(options=win_options)\n\ntestCases\\conftest.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nvenv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py:45: in __init__\n    super().__init__(\nvenv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py:51: in __init__\n    self.service.path = DriverFinder.get_path(self.service, options)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nservice = <selenium.webdriver.chrome.service.Service object at 0x000002DBD6F5BE50>\noptions = <selenium.webdriver.chrome.options.Options object at 0x000002DBD9746CD0>\n\n    @staticmethod\n    def get_path(service: Service, options: BaseOptions) -> str:\n        path = service.path\n        try:\n            path = SeleniumManager().driver_location(options) if path is None else path\n        except Exception as err:\n            msg = f\"Unable to obtain driver for {options.capabilities['browserName']} using Selenium Manager.\"\n>           raise NoSuchDriverException(msg) from err\nE           selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for chrome using Selenium Manager.; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py:41: NoSuchDriverException"}, "start": 1691723685865, "stop": 1691723685865, "uuid": "c15b4dc7-3775-4c64-9add-55d8a5f03e6e", "historyId": "37518f13d7d322a452692dcf9fa516e6", "testCaseId": "37518f13d7d322a452692dcf9fa516e6", "fullName": "testCases.test_login.Test_Login#test_page_title_001", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "This is story #01"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "DESKTOP-HFVASSE"}, {"name": "thread", "value": "22384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}], "links": [{"type": "issue", "url": "IssueNo-225", "name": "IssueNo-225"}, {"type": "link", "url": "https://opensource-demo.orangehrmlive.com/", "name": "https://opensource-demo.orangehrmlive.com/"}]}